.github/workflows/scanner-build.yaml

```diff
@@ -61,6 +61,8 @@ jobs:
         QUAY_RHACS_ENG_RW_PASSWORD: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
         QUAY_STACKROX_IO_RW_USERNAME: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
         QUAY_STACKROX_IO_RW_PASSWORD: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
+        # If true, scanner-db image will ship with the DB init bundle.
+        SCANNER_DB_INIT_BUNDLE_ENABLED: ${{ contains(github.event.pull_request.labels.*.name, 'pr-scanner-db-init-bundle') }}
```

Comment:
Is the (current) expectation to not offer the init bundles on tags (e.g., nightlies or releases)?

Reply:
@BradLugo I am not sure what the impact of adding this to nightlies and release builds is -- I don't have visibility into how this would affect downstream builds, so I am being defensive.

I definitely want this be "true" for builds in master. Any recommendations? I can follow up on another PR if better.

Reply:
I am going to follow up in a separate PR to enable this for all builds. Resolving.